recipes/R/Retail - GradientBoosting/R/applicationScorer.R

```diff
@@ -97,7 +97,11 @@ applicationScorer <- setRefClass("applicationScorer",
         mutate(prediction = pred,
                store = as.integer(store),
                date = as.character(date))
-      
+
+      tenantId = configurationJSON$tenantId
+      colnames(output_df) <- paste(tenantId, colnames(output_df), sep = ".")
```

Comment:
just a quick question. I seem to see that model.rds does not save the tenant id, which is good; is there any way that this action could trigger the format to be <tenant_id>.<tenant_id>.<field> ?

Reply:
No. TenantId is removed before applying training and saving the model. This is just returning back after scoring in order to write to the dataset that is based on 0.999 XMD schema. It can't trigger something like <tenant_id>.<tenant_id>.